data/json/items/ammo.json

```diff
@@ -1327,5 +1327,25 @@
     "count": 50,
     "ammo_type": "tinder",
     "effects": [ "NEVER_MISFIRES" ]
+  },
+  {
+    "type": "AMMO",
+    "id": "rpg_dice",
+    "price": 10,
+    "name": "rpg dice",
```

Comment:
You need a `"name_plural", or else multiples will show up as `x rpg dices`.

Reply:
okay so (I really don't know how this works) should I end this one and make a new one with that in?

Reply:
No, you make the change, commit it, and push it to this branch.

Reply:
also, the singular is "die"